t/27-consoles-vnc.t

```diff
@@ -365,4 +367,28 @@ subtest 'server initialization' => sub {
     is_deeply \@printed, \@expected, 'pixel format and encodings replied' or diag explain \@printed;
 };
 
+subtest 'login on real VNC server via vnctest, request and receive frame buffer' => sub {
+    # This test is using `vnctest` so this script is covered as well. Note that the `vnctest` script has mainly been added to be able to run our VNC client
+    # code manually against a real VNC server (which can sometimes be useful).
+
+    my $display = 20 // $ENV{VNC_TEST_DISPLAY};
```

Comment:
What's this supposed to do? This is always 20, no? Did you mean the reverse?

Reply:
Yes, I meant the reverse. Good catch.